src/components/fhir/FhirAuthoriser.tsx

```diff
@@ -15,7 +15,7 @@ const FhirAuthoriser: FC = () => {
   useEffect(() => {
     SMART.authorize({
       iss: FHIR_URL,
-      redirectUri: "./new_report",
+      redirectUri: "#/new_report",
```

Comment:
Is this needed to satisfy the requirements of the `Hash Router`?

Reply:
yep!